test/compile_errors.zig

```diff
@@ -7419,7 +7435,7 @@ pub fn addCases(cases: *tests.CompileErrorContext) void {
     });
 
     cases.add( // fixed bug #2032
-        "compile diagnostic string for top level decl type",
+    "compile diagnostic string for top level decl type",
```

Comment:
what's going on with the indentation here?

Reply:
I think that's related to the recent zig fmt changes. @LakeByTheWoods is this a bug?

Reply:
Ahhh, I new there had to be at least *one* regression. Yeah it's a bug. Would be fixed with the fix for #5343 if we want that as well.